### PR TITLE
Deploy debian packages to bintray

### PR DESCRIPTION
**Pull request by jlongever**
@RackHD/corecommitters

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,6 +24,25 @@ after_success:
...
+ - git log -n 1 --pretty=format:%h.%ai.%s > commitstring.txt
+ - debuild --no-lintian --no-tgz-check -us -uc
+ - popd
+ - cp -a *.deb deb/
+
+deploy: 
+  provider: bintray
+  file: .bintray.json
+  user: $BINTRAY_USER
+  key: $BINTRAY_KEY
```
Comment (yyscamper) [inline .travis.yml]: Where is the USRER/KEY from?

Comment (heckj) [inline .travis.yml]: These are provided through the travis ENV variables set up per repository. For an example, you can see the configuration control (as an admin on the repo) for the `on-imagebuilder` repo at https://travis-ci.org/RackHD/on-imagebuilder/settings.

**Comment by heckj** [comment]
we probably want to manually update the debian changelog string to have a more recently version than the defaults in there now (which are legacy from months ago), or do a similar `dch` command for force a local version to be clear on when it was built.

Also - will this build on each PR, or will it only build when merged into master?

**Comment by jlongever** [comment]
@heckj should only deploy when changes are committed to the master branch.

**Comment by jlongever** [comment]
@heckj I'll update to run dch.

File: .travis.yml (hunk 1 above)
Comment (jlongever) [inline .travis.yml]: They are set as travis-ci variables.

**Comment by jlongever** [comment]
@heckj for the moment i'll run dch to update the changelog with the branch (should just be master) and commit log.  I assume we don't want to deploy new .debs for every master commit?

**Comment by heckj** [comment]
Works for me!

**Comment by benbp** [comment]
:+1:

**Comment by zyoung51** [comment]
+1

**Pull request merged**